tabs/configuration.js

```diff
@@ -436,6 +451,104 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
         $('input[name="currentoffset"]').val(BF_CONFIG.currentoffset);
         $('input[name="multiwiicurrentoutput"]').prop('checked', MISC.multiwiicurrentoutput);
 
+        var escProtocols = {
+            0: {
+                name: "STANDARD",
+                defaultRate: 400,
+                rates: {
+                    50: "50Hz",
+                    400: "400Hz"
+                }
+            },
+            1: {
+                name: "ONESHOT125",
+                defaultRate: 1000,
+                rates: {
+                    400: "400Hz",
+                    1000: "1kHz",
+                    2000: "2kHz",
+                    4000: "4kHz"
+                }
+            },
+            2: {
+                name: "ONESHOT42",
+                defaultRate: 2000,
+                rates: {
+                    400: "400Hz",
+                    1000: "1kHz",
+                    2000: "2kHz",
+                    4000: "4kHz",
+                    8000: "8kHz"
+                }
+            },
+            3: {
+                name: "MULTISHOT",
+                defaultRate: 2000,
+                rates: {
+                    400: "400Hz",
+                    1000: "1kHz",
+                    2000: "2kHz",
+                    4000: "4kHz",
+                    8000: "8kHz"
+                }
+            },
+            4: {
+                name: "BRUSHED",
+                defaultRate: 8000,
+                rates: {
+                    500: "500Hz",
```

Comment:
I think brushed shouldn't have anything but 8/16/32 khz. 4kHz and below will make motors make very annoying sound when working.